docs/server-versions-dependencies.md

```diff
@@ -10,6 +10,23 @@ This page details some of the version specific requirements and dependencies nee
 
 All versions of Temporal require that [Go v1.15+](https://golang.org/dl/) is installed in the environment.
 
+## Temporal Versioning and Support Policy
+
+All of Temporal's releases abide by [semver](https://semver.org/), but here we detail the precise upgrade paths and support we offer, starting from Temporal `v1.7.0`:
+
+- We define maintenance support as continuing to release critical bugfixes — related to security, preventing data loss, reliability — for previously published minor and major versions, whenever they are found.
+- **For Temporal Server OSS**
+    - We will publish incremental upgrade guides for each minor and major version
+        - This also includes specifying what dependency upgrades we test for (e.g. from Cassandra 3.0 to 3.11)
```

Comment:
As my recommendation above suggest, drop this bullet and just combine with the sentence above.

Reply:
```suggestion
```